doc/source/whatsnew/v0.23.0.txt

```diff
@@ -214,11 +214,21 @@ Please note that the string `index` is not supported with the round trip format,
    :okwarning:
 
    df.index.name = 'index'
-   df.to_json('test.json', orient='table')
-   new_df = pd.read_json('test.json', orient='table')
+
+   test_json_buf = io.StringIO()
```

Comment:
Ideally, a user reading this doesn't have to care about any of the stuff with the IO buffer. They only care about writing and reading the dataframe.

I think it's best to leave the `df.to_json` and `new_df = ` as it was previosly.

Then we can add a new `ipython` block afterwards to remove the file.

```
.. ipython:: python
   :suppress:

   import os
   os.remove('test.json')
```